solver/Cargo.toml

```diff
@@ -31,6 +33,7 @@ tracing = { workspace = true }
 lru = "0.12.3"
 rand = { workspace = true }
 num-rational = { workspace = true }
+num-traits = "*"
```

Comment:
Not sure this one is actually needed (if we remove the use of `zero()`

Reply:
Fix: 28006c9b734f50e6491dd1d36d3c333d52fe5057 "remove num-traits dependency"